uber/templates/static_views/stafferComps.html

```diff
@@ -11,7 +11,7 @@ <h3 align="center"> What You Get For Volunteering </h3>
     <li> Working at least 6 hours gets you a {{ c.EVENT_NAME }} t-shirt. <br/><br/> </li>
     <li> Working at least 12 hours gets you food in addition to the shirt.  We have professional chefs preparing 3 meals per day throughout the event. <br/><br/> </li>
     <li> Working at least 18 hours gets you a complementary Staff badge for next year's {{ c.EVENT_NAME }}, which will make you eligible for space in one of our staff hotel rooms. <br/><br/> </li>
-    <li> Working at least 24 hours gets you a refund on your badge for this year. <br/><br/> </li>
+    <li> Working at least {{ c.HOURS_FOR_REFUND }} hours gets you a refund on your badge for this year. <br/><br/> </li>
```

Comment:
Surprisingly, this was the only other place in the code where we referred to the number 24 w.r.t. hour requirements.